src/appleseed/renderer/kernel/lighting/directlightingintegrator.cpp

```diff
@@ -606,31 +606,28 @@ void DirectLightingIntegrator::add_emitting_triangle_sample_contribution(
     cos_on *= rcp_sample_distance;
     incoming *= rcp_sample_distance;
     
-    // Decide whether the contributon from this light is significant or not.
+    // Compute the approximate contribution.
     const double approximate_contribution = 
-        cos_on * 
+        (cos_on * 
         rcp_sample_square_distance * 
-        edf->get_max_contribution();
+        edf->get_max_contribution()) 
+        / sample.m_triangle->m_rcp_area;
```

Comment:
Yes, i used rcp because there there was not an area property in the triangle and i didn't want to add one. Should i add it?